src/components/Wizard/CreateVmWizard/CreateVmWizard.js

```diff
@@ -116,9 +116,10 @@ export class CreateVmWizard extends React.Component {
 
   lastStepReached = () => this.state.activeStepIndex === this.getLastStepIndex();
 
-  onStepDataChanged = (tabKey, value, valid) => {
+  onStepDataChanged = (tabKey, value, valid, editing = false) => {
     const validatedTabData = validateTabData(tabKey, value, valid);
     this.safeSetState(state => ({
+      editing,
```

Comment:
I think the tab should merely request to lock the createVmWizard and not take Wizard state into account. This is very useful when using asynchronous actions.

I suggest something like:

```js
{
  stepData: {
    [tabKey]: {
      lockStep
    }
  }
}

```

Reply:
done, `lockStep` is under `[tabKey]` now